.github/workflows/doc-linting.yml

```diff
@@ -1,10 +1,48 @@
 name: Lint and check documentation
-on: [push]
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Comment:
shouldn't we use the autofix on all branches?

Reply:
Thanks, good point! 👍🏻